picard/script/functions.py

```diff
@@ -680,60 +680,97 @@ def func_ne(parser, x, y):
         return ""
 
 
+def _cmp(op, x, y, _type):
+    """Compare x vs y using op method and specified _type
+       op is expected to be a method from operator module
+    """
+    if not _type:
+        _type = 'auto'
+    _typer = None
+    if _type == 'auto':
+        _type = 'text'
+        for _test_type in [int, float]:
```

Comment:
Just a minor thing, but a tuple (to preserver order) would be better than a list
```suggestion
        for _test_type in (int, float):
```